spec/helpers/ems_cloud_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+
+describe EmsCloudHelper do
+  before do
+    controller.send(:extend, EmsCloudHelper)
+    self.class.send(:include, EmsCloudHelper)
+  end
+
+  context "textual_instances" do
+    def role_allows(_)
+      true
+    end
```

Comment:
This is a way to stub ```role_allows``` and make it return ```true```
The usual way of stubbing -
```
controller.stub(:role_allows).and_return(true)
```
does not seem to work.
There is a similar usage in <a href=https://github.com/ManageIQ/manageiq/blame/master/spec/helpers/application_helper_spec.rb#L1424>application_helper_spec</a>